package.json

```diff
@@ -85,9 +85,9 @@
     "stylelint-processor-styled-components": "1.3.1"
   },
   "peerDependencies": {
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "react-native": "^0.50.3",
+    "react": "^16.3.1",
```

Comment:
Hotrod and rodimus are still on 16.2. So I would recommend not to add peerDependencies for react to be ^16.3.1